new-backend/.env

```diff
@@ -213,3 +213,6 @@ OPENAPI_ENABLE_RESPONSE_VALIDATION=false
 #ANALYTICS_TYPE="matomo" 
 #ANALYTICS_OPTION_TRACKERURL=https://xxxxxxxxxx.matomo.cloud
 #ANALYTICS_OPTION_SITEID=4
+
+# FIR_REALESTATEOWNERREPORT_URL=https://fastighetsrapport-utv.domain.se/Report/fastighetsforteckning/xlsx
+# FIR_TEMP_OUTPUT_DIR=/temp/
```

Comment:
Sorry to by picky but can you add some comments about the recent additions in .env? Take a look at the sections above and do something similar. :)